tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -103,11 +103,9 @@ def update_encrypted_fields(self, is_encrypted):
                 if type(field) == TransformField and "is_encrypted" in field.kwargs:
                     field.kwargs['is_encrypted'] = is_encrypted
 
-def contains_encrypted_indicator(line, encryption_field):
-    """Determine if line contains encryption indicator."""
-    if encryption_field is not None:
-        return encryption_field.parse_value(line) == "E"
-    return False
+def is_string_field_valid(val, field_length):
```

Comment:
due to circular imports, i had to move some functions like this (pure, validator-ish functions that don't import from our application code) from `util` to `validators`, namely `value_is_empty`.

i'd recommend either moving this for consistency, or perhaps even parameterizing the `empty_values` array in `value_is_empty` to include a `0`-fill case since the function is so similar

Reply:
Going to wait until your work (#2748) merges, that way I can update based off of this feedback.